test/unit/api/ws/workers/peersUpdateRules.js

```diff
@@ -98,11 +101,41 @@ describe('PeersUpdateRules', function () {
 			expect(connectionsTable.connectionIdToNonceMap).to.have.property(validConnectionId).equal(validPeer.nonce);
 		});
 
-		it('should remove added entries from connectionsTable after receiving an error from server', function (done) {
+		it('should return the error from server when invoked with valid arguments and received error code', function (done) {
```

Comment:
`should return an error from server when invoked with valid arguments and received error code`